scss/style.scss

```
@@ -0,0 +1,533 @@
+@font-face {
+  font-family: "MyFont";
+  src: url("../font/MyFont.eot?s18mh3");
+  src: url("../fonts/MyFont.eot?s18mh3#iefix") format("embedded-opentype"),
+    url("../font/MyFont.ttf?s18mh3") format("truetype"),
+    url("../font/MyFont.woff?s18mh3") format("woff"),
+    url("../font/MyFont.svg?s18mh3#icomoon") format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"],
+[class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: "MyFont" !important;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-check-circle:before {
+  content: "\e900";
+}
+.icon-chevron-down:before {
+  content: "\e901";
+}
+.icon-facebook-f:before {
+  content: "\e902";
+}
+.icon-linkedin:before {
+  content: "\e903";
+}
+.icon-pinterest:before {
+  content: "\e904";
+}
+.icon-quote-left:before {
+  content: "\e905";
+}
+.icon-search:before {
+  content: "\e906";
+}
+.icon-thumbs-up:before {
+  content: "\e907";
+}
+.icon-twitter:before {
+  content: "\e908";
+}
+.icon-youtube:before {
+  content: "\e909";
+}
+$aleo: "Aleo", serif;
+$josefin: "Josefin Sans", sans-serif;
+$white: #fff;
+$grey: #999999;
+$blue: #5373fd;
+@mixin back {
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+@mixin h2 {
+  font-weight: normal;
+  font-size: 36px;
+  line-height: 40px;
+  text-transform: capitalize;
+}
+@mixin but {
+  min-width: 213px;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 20px;
+  letter-spacing: 0.07em;
+  text-transform: uppercase;
+  color: $white;
+  padding: 20px 32px;
+  background: url(../img/ButtonBG.png) no-repeat left center, $blue;
+  border: 1px solid transparent;
+  cursor: pointer;
+  &:hover {
+    background-size: 85%;
+    background-position-x: 50px;
+  }
+}
+* {
+  margin: 0;
+  padding: 0;
+  transition: color 0.2s, border 0.2s, background 0.2s, display 0.2s;
+  box-sizing: border-box;
+}
+body {
+  font-family: $aleo;
+  font-weight: 300;
+  color: $white;
+}
+.header {
+  height: 90vh;
```

```suggestion
  min-height: 90vh;
```
шоб не зменшувалось більше, ніж потрібно.. але не знаю, як було задумано